py_zipkin/zipkin.py

```diff
@@ -402,14 +382,16 @@ def stop(self, _exc_type=None, _exc_value=None, _exc_traceback=None):
             if annotation in self.annotation_filter:
                 self.annotations.setdefault(annotation, timestamp)
 
-        self.log_handler.store_local_span(
-            span_name=self.span_name,
-            service_name=self.service_name,
-            annotations=self.annotations,
-            binary_annotations=self.binary_annotations,
-            sa_endpoint=self.sa_endpoint,
-            span_id=self.zipkin_attrs.span_id,
-        )
+        self._span_store.push({
+            'trace_id': self.zipkin_attrs.trace_id,
+            'span_name': self.span_name,
+            'service_name': self.service_name,
+            'parent_span_id': self.zipkin_attrs.parent_span_id,
+            'span_id': self.zipkin_attrs.span_id,
+            'annotations': self.annotations,
+            'binary_annotations': self.binary_annotations,
+            'sa_endpoint': self.sa_endpoint,
```

Comment:
I think this is the same comment as in the other PR, a span type would be cleaner i think

Reply:
I agree. I just want to keep the 2 PR separated, so I'll add the Span type in the other one.